site/en/guides/build/features/jam-detection.md

```diff
@@ -398,7 +398,7 @@ JamDetection:RssiThreshold = -45
 ```
 
 For more information on `wpantund` properties, see the [`wpantund` GitHub
-repository](https://github.com/openthread/wpantund/tree/master/src/wpantund/wpan-properties.h).
+repository](https://github.com/openthread/wpantund/tree/main/src/wpantund/wpan-properties.h).
```

Comment:
Actually, it looks like wpantund hasn't been switched from master to main (link doesn't work now), so let's keep this as master.  So on DevSite it should be "master" for the wpantund repo.

Reply:
Done.